rootfs/docker-entrypoint-initdb.d/003_restore_from_backup.sh

```diff
@@ -55,3 +55,6 @@ fi
 # ensure $PGDATA has the right permissions
 chown -R postgres:postgres "$PGDATA"
 chmod 0700 "$PGDATA"
+
+# Run periodic backups in the background
+gosu postgres backup &
```

Comment:
My only question woyld be re: placement of this.  A script whose job is "restore" doesn't seem the right place to kick off backups.  Maybe this is deserving of its own script (even if it's a one-liner) in `rootfs/docker-entrypoint-initdb.d`?